src/bootstrap/container.ts

```diff
@@ -1,24 +1,23 @@
 import Bottle from 'bottlejs';
-import { ClientService } from '../services/client.service';
-import { HttpService } from '../services/http.service';
-import { PluginService } from '../services/plugin.service';
-import { MessageService } from '../services/message.service';
 import { ChannelService } from '../services/channel.service';
 import { ClassService } from '../services/class.service';
+import { ClientService } from '../services/client.service';
+import { ControllerService } from '../services/controller.service';
+import { GameLeaderboardService } from '../services/gameleaderboard.service';
 import { GuildService } from '../services/guild.service';
 import { HandlerService } from '../services/handler.service';
+import { HttpService } from '../services/http.service';
 import { JobService } from '../services/job.service';
-import { StoreService } from '../services/store.service';
-import { ModService } from '../services/moderation.service';
-import { StorageService } from '../services/storage.service';
 import { LoggerService } from '../services/logger.service';
+import { MessageService } from '../services/message.service';
+import { ModService } from '../services/moderation.service';
+import { PluginService } from '../services/plugin.service';
+import { PointService } from '../services/point.service';
 import { RoleService } from '../services/role.service';
-import { PollService } from '../services/poll.service';
-import { WarningService } from '../services/warning.service';
-import { GameLeaderboardService } from '../services/gameleaderboard.service';
+import { StorageService } from '../services/storage.service';
+import { StoreService } from '../services/store.service';
 import { UserService } from '../services/user.service';
-import { ControllerService } from '../services/controller.service';
-import { PointService } from '../services/point.service';
+import { WarningService } from '../services/warning.service';
```

Comment:
This is just my import sorter going ham ig

Reply:
maybe just to practice, you can use `git add -p` next time and then not select changes like this